src/main/resources/application-local-docker.yml

```diff
@@ -47,6 +47,7 @@ authorisation:
 hmpps.sqs:
   provider: localstack
   localstackUrl: http://local-stack-aws:4566
+  useWebToken: false
```

Comment:
what's this for?

Reply:
Version 3 of hmpps.sqs changed useWebToken to default to true, therefore we need to set it to false when we using localstack. [Breaking changes](https://github.com/ministryofjustice/hmpps-spring-boot-sqs/blob/main/release-notes/3.x.md#Breaking-changes)

Reply:
Great